src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -261,10 +268,11 @@ Namespace Microsoft.CodeAnalysis.Semantics
             limitValue As IOperation,
             stepValue As IOperation) As ImmutableArray(Of IOperation)
             Dim statements As ArrayBuilder(Of IOperation) = ArrayBuilder(Of IOperation).GetInstance()
+            Dim isImplicit As Boolean = controlVariable.WasCompilerGenerated
```

Comment:
>Dim isImplicit As Boolean = controlVariable.WasCompilerGenerated [](start = 12, length = 64)

This should probably be moved inside the ```if``` below, could be a NullReferenceException otherwise.